<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_sign_up</class>
 <widget class="QDialog" name="Dialog_sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_role">
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Роль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_fio">
       <property name="placeholderText">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_login">
       <property name="placeholderText">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="inputMask">
        <string>+7-999-999-99-99</string>
       </property>
       <property name="placeholderText">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_file_dialog">
       <property name="text">
        <string>Картинка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_req_sign_up">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
